Makefile.vars

```diff
@@ -143,6 +143,7 @@ PYBABEL := $(ROOT)/build/env/bin/pybabel
 # Path to files for pip requirements
 ##############################
 REQUIREMENT_FILE := $(ROOT)/desktop/core/requirements.txt
+REQUIREMENT_PPC64LE_FILE := $(ROOT)/desktop/core/requirements_ppc64le.txt
```

Comment:
Shouldn't the new requirements file be part of this change?

Reply:
It has been submitted to internal GitHub.

Reply:
@ranade1 @JohanAhlen  
We were testing build on  ppc64le architecture and build is failing due to missing file desktop/core/requirements_ppc64le.txt .
Also build passed with desktop/core/requirements.txt. So is following change required in makefile which is merged in this PR
@if [[ "ppc64le" == $(PPC64LE) ]]; then \
	  echo '--- Installing $(REQUIREMENT_PPC64LE_FILE) into virtual-env via $(ENV_PIP)'; \
	  $(ENV_PIP) install -r $(REQUIREMENT_PPC64LE_FILE); \
	  echo '--- Finished $(REQUIREMENT_PPC64LE_FILE) into virtual-env'; \

If above change is required we will need desktop/core/requirements_ppc64le.txt.

Kindly suggest